addons/mail/static/src/core/common/record.js

```diff
@@ -79,13 +414,51 @@ export class Record {
         }
         return record;
     }
-
+    /**
+     * @template {keyof import("model ").Models} M
+     * @param {M} modelName
+     * @returns {import("models").Models[M]}
+     */
+    static one(modelName) {
+        return ONE_SYM;
+    }
+    /**
+     * @template {keyof import("model").Models} M
+     * @param {M} modelName
+     * @returns {import("models").Collection<import("models").Models[M]>["List"]}
+     */
+    static List(modelName) {
+        return MANY_LIST_SYM;
+    }
+    /**
+     * @template {keyof import("model").Models} M
+     * @param {M} modelName
+     * @returns {import("models").Collection<import("models").Models[M]>["Set"]}
+     */
+    static Set(modelName) {
+        return MANY_SET_SYM;
+    }
     /**
      * @param {Object} data
      * @returns {Record}
      */
     static insert(data) {}
 
+    /**
+     * Raw relational values of the record, each of which contains object id(s)
+     * rather than the record(s). This allows data in store and models being normalized,
+     * which eases handling relations notably in when a record gets deleted.
+     *
+     * @type {Map<string, string|RecordList|RecordSet>}
+     */
+    __rels__ = new Map();
+    /**
+     * Track inverse relations of current record. key is inverse field name @see Record.__invs__
+     * and value contains local ids of records that have relational value on current record on
+     * inverse field name. Note that you can look up `Record.__invs__` for inverse of inverse
+     * field name.
+     */
```

Comment:
Comment is incorrect. There's no longer `Record.__invs__` (it's a residue from an old implem)